ntbs-service/Migrations/20190930103847_MergeRiskFactorsIntoSameModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;

```

Comment:
I did some refactoring of this, as in the audit logs all risk factors just appear with model name "SocialRiskFactor", which is actually the entity that owns the individual risk factors for Homelessness, Drugs Imprisonment, which is itself owned by Notification. Before adding "Type" there was no way in the audit logs to distinguish between them as they all looked identical.

Now that they have a type column to distinguish them, I think it would be nicer if they were all mapped to the same table (separate from SocialRiskFactor itself, which is the owner), but EF didn't like this somehow. Maybe it's possible to make work, but I think it was out of scope for this ticket.